docs/user/pings/index.md

```diff
@@ -46,8 +46,8 @@ Optional fields are marked accordingly.
 | `app_display_version` | String | The user-visible version string (e.g. "1.0.3") |
 | `architecture` | String | The architecture of the device (e.g. "arm", "x86") |
 | `client_id` | UUID |  *Optional* A UUID identifying a profile and allowing user-oriented correlation of data |
-| `device_manufacturer` | String | The manufacturer of the device |
-| `device_model` | String | The model name of the device. On Android, this is [`Build.MODEL`], the user-visible name of the device. |
+| `device_manufacturer` | String | *Optional* The manufacturer of the device |
```

Comment:
Would you kindly also mention the new behaviour [here](https://github.com/mozilla/glean/blob/4311592bf451259d0c139fa64cf0eec3945de091/glean-core/metrics.yaml#L77-L107)?